### PR TITLE
Convert to swift 5 

### PR DESCRIPTION
**Pull request by nmatijevic1**
Changes in .project file and scheme

### CODE REVIEW

File: Example/Tests/KeyTests.swift [hunk 1]
```diff
@@ -107,7 +107,7 @@ final class KeyTests: XCTestCase {
 
     // when
     let publicKey = try? privateKey.publicKeyRepresentation()
-    let publicKeyPEM = try? publicKey?.convertedToPEM()
+    let publicKeyPEM = ((try? publicKey?.convertedToPEM()) as String??)
```
Comment (thehungrydev) [inline Example/Tests/KeyTests.swift]: This one seems odd, does it really need double parentheses?

File: Krypt/Source/AES256.swift [hunk 2]
```diff
@@ -72,8 +72,8 @@ private extension AES256 {
   /// - Returns: random data
   static func randomData(count: Int) -> Data {
     var data = Data(count: count)
-    let status = data.withUnsafeMutableBytes {
-      SecRandomCopyBytes(kSecRandomDefault, count, $0)
+    let status = data.withUnsafeMutableBytes { ptr in
+        SecRandomCopyBytes(kSecRandomDefault, count, (ptr.baseAddress?.assumingMemoryBound(to: UnsafeRawBufferPointer.self))!)
```
Comment (thehungrydev) [inline Krypt/Source/AES256.swift]: Should avoid force unwrapping. 

Also I see that this function is pretty much a duplicate of the one in `MedStrickerEncryptionTests`. Maybe we could extract this to a place where they can be reused.

File: Example/Tests/MedStickerEncryptionTests.swift [hunk 3]
```diff
@@ -207,8 +207,9 @@ final class MedStickerEncryptionTests: XCTestCase {
 
   private func randomData(count: Int) -> Data {
     var data = Data(count: count)
-    data.withUnsafeMutableBytes {
-      SecRandomCopyBytes(kSecRandomDefault, count, $0)
+    data.withUnsafeMutableBytes { ptr in
+      guard let pointer = ptr.baseAddress?.assumingMemoryBound(to: UnsafeRawBufferPointer.self) else { return }
```
Comment (thehungrydev) [inline Example/Tests/MedStickerEncryptionTests.swift]: I'm seeing a potential issue where we just return and don't generate random bytes, could be quite dangerous.

File: Krypt/Source/MedStickerEncryption.swift [hunk 4]
```diff
@@ -112,7 +112,7 @@ public struct MedStickerEncryption {
     let combinedBytes = [attr.key, attr.iv, salt]
       .compactMap([UInt8].init)
       .reduce([UInt8](), +)
-    let combinedData = Data(bytes: combinedBytes)
+    let combinedData = Data( combinedBytes)
```
Comment (thehungrydev) [inline Krypt/Source/MedStickerEncryption.swift]: SwiftFormat not running

File: Krypt/Source/SHA256.swift [hunk 5]
```diff
@@ -11,9 +11,10 @@ import Foundation
 public struct SHA256 {
   public static func digest(_ data: Data) -> Data {
     var hash = [UInt8](repeating: 0, count: Int(CC_SHA256_DIGEST_LENGTH))
-    data.withUnsafeBytes {
-      _ = CC_SHA256($0, CC_LONG(data.count), &hash)
+    data.withUnsafeBytes { ptr in
+        guard let pointer = ptr.baseAddress?.assumingMemoryBound(to: UnsafeRawBufferPointer.self ) else { return }
```
Comment (thehungrydev) [inline Krypt/Source/SHA256.swift]: This one also seems to return just bytes of 0 instead of the hash in case of failed pointer conversion.

**Review by markotect — CHANGES_REQUESTED** [review]

File: Example/Tests/MedStickerEncryptionTests.swift [hunk 6]
```diff
@@ -154,7 +154,7 @@ final class MedStickerEncryptionTests: XCTestCase {
     )
 
     // when
-    let signature = MedStickerEncryption.accessSignature(attr: cipherAttr, salt: salt)
+    let signature = MedStickerEncryption.accessSignature(attr: cipherAttr, salt: salt)!
```
Comment (thehungrydev) [inline Example/Tests/MedStickerEncryptionTests.swift]: Should keep consistent probably with where you force unwrap. e.g. test below.

**Review by markotect — CHANGES_REQUESTED** [review]

**Review by markotect — APPROVED** [review]

**Pull request merged**